pmd-visualforce/src/test/java/net/sourceforge/pmd/lang/vf/RuleSetFactoryTest.java

```diff
@@ -5,7 +5,13 @@
 package net.sourceforge.pmd.lang.vf;
 
 import net.sourceforge.pmd.AbstractRuleSetFactoryTest;
+import net.sourceforge.pmd.lang.apex.rule.ApexXPathRule;
 
 public class RuleSetFactoryTest extends AbstractRuleSetFactoryTest {
-    // no additional tests
+    public RuleSetFactoryTest() {
+        super();
+        // Copied from net.sourceforge.pmd.lang.apex.RuleSetFactoryTest
+        // Apex rules are found in the classpath because this module has a dependency on pmd-apex
```

Comment:
I think, we should fix/improve AbstractRuleSetFactoryTest. I guess, both apex and visualforce rules are now tested, which is unnecessary.

Reply:
I added the ability to exclude Languages from the AbstractRuleSetFactoryTest.